src/Bicep.Core/Diagnostics/DiagnosticBuilder.cs

```diff
@@ -809,6 +809,21 @@ public ErrorDiagnostic ArgumentCountMismatch(int argumentCount, int mininumArgum
                 TextSpan,
                 "BCP141",
                 "The expression cannot be used as a decorator as it is not callable.");
+
+            public ErrorDiagnostic TooManyPropertyLoops() => new(
+                TextSpan,
+                "BCP142",
+                "Property value for-expressions cannot be nested.");
+
+            public ErrorDiagnostic ExpressionedPropertiesNotAllowedWithLoops() => new(

```

Comment:
Could you add an example of this to one of the baseline files?

Reply:
Ugh forgot about that one.

---
In reply to: [582079696](https://github.com/Azure/bicep/pull/1521#discussion_r582079696) [](ancestors = 582079696)

Reply:
Done.